examples/integrated-flows-examples/src/verification/app.ts

```diff
@@ -18,9 +18,9 @@ dotenv.config();
 
   enum VerificationMethods {
     SMS ='sms',
-    CALLOUT = 'callout',
+    PHONE_CALL = 'phone call',
```

Comment:
Underlying identifiers (protocol layer) are still 'callout' and 'seamless' (see https://gitlab.com/sinch/sinch-projects/enterprise-and-messaging/documentation/developer-experience/oas-documentation/-/blob/master/services/verification-oas/verification.yaml?ref_type=heads#L776)
Only the the _presenter_ terms was changed (no underlying protocol breaking change)

So we could imagine having:
```
/**
 * @deprecated The identifier should not be used and prefer PHONE_CALL usage
 */
CALLOUT = 'callout',
PHONE_CALL = 'callout',
```
(same for `SEAMLESS`)
So we avoid a hard break change and are able to provide a smooth transition period to users until the identifiers will be really removed (and will release SDK with a major version change i.e. : 2.0)

Reply:
In this small program, I don't consider it as a breaking change but rather a bug fix to present the same terminology as in the documentation to the end user when the program will list the available verification methods. WDYT?

Reply:
My bad,
I was not focused enough and did not take attention it is a sample app
I was thinking it was the "official" supported values for SDK
Sorry you should ignore this thread...